docs/rules/no-extra-parens.md

```diff
@@ -36,6 +36,12 @@ a = (b * c);
 
 (a * b) + c;
 
+for (a in (b, c));
+
+for (a in (b));
+
+for (a of (b);
```

Comment:
Missing a `)` here?

Reply:
@j-f1 yup just fixed that! Pushed a little too fast :)